Extension/src/common.ts

```diff
@@ -86,24 +87,33 @@ export async function ensureBuildTaskExists(taskName: string): Promise<void> {
         rawTasksJson.tasks = new Array();
     }
     // Find or create the task which should be created based on the selected "debug configuration".
-    let selectedTask: vscode.Task = rawTasksJson.tasks.find(task => task.label && task.label === task);
+    let selectedTask: vscode.Task | undefined = rawTasksJson.tasks.find((task: any) => task.label && task.label === task);
     if (selectedTask) {
         return;
     }
 
     const buildTasks: vscode.Task[] = await getBuildTasks(false);
     selectedTask = buildTasks.find(task => task.name === taskName);
     console.assert(selectedTask);
+    if (!selectedTask) {
+        throw new Error("Failed to get selectedTask in ensureBuildTaskExists()");
```

Comment:
do these Error strings need to be localized?

Reply:
The only 'throw new Error's that I've added are for scenarios that should not occur, so shouldn't generally be diagnosable to the user.  My opinion is no, we shouldn't localize them.  In fact, the more of these that can be removed by designing around them, the better.